api/bazel/BUILD

```diff
@@ -15,3 +18,14 @@ go_proto_compiler(
     valid_archive = False,
     visibility = ["//visibility:public"],
 )
+
+write_json(
```

Comment:
Prefer that we rename this to something like `json_data` or something. Basically, Bazel targets are nodes in a dependency graph, not really imperative commands. That said, `write_file` is also badly named in Bazel itself.

Reply:
sure, ill update

Reply:
updated